sdk/keyvault/keyvault-secrets/test/internal/serviceVersionParameter.spec.ts

```diff
@@ -63,11 +63,6 @@ describe("The Secrets client should set the serviceVersion", () => {
   it("it should allow us to specify an API version from a specific set of versions", async function() {
     const versions: ApIVersions[] = ["7.0", "7.1"];
     for (const serviceVersion in versions) {
-      const credential = await new ClientSecretCredential(
```

Comment:
@sadasant I think it was a mistake to redefine `credential` here, no?

Reply:
Looks like that is true! Thank you 🌞

Reply:
![image](https://user-images.githubusercontent.com/417016/101941250-b7944900-3bb5-11eb-8637-9c710b4fe4b1.png)
but why

Reply:
I think I sent a meme here by mistake. I'll point out where I intended to send it. Nothing big! Just trying to be funny 😛